src/App.js

```diff
@@ -1,53 +1,64 @@
-import { CiWarning,CiMenuFries,CiUser,CiShop } from "react-icons/ci";
+import { CiWarning, CiMenuFries, CiUser, CiShop } from "react-icons/ci";
 import { HiOutlineShoppingBag } from "react-icons/hi";
+// import section6 from "./componenets/Homepage/Section6/section6";
+// import homePage from "./componenets/HomePage/homePage";
+import HomePage from "./componenets/Homepage/HomePage"
 
 import './App.css';
 
 function App() {
   return (
-   <div>
-      <nav class="main-nav">
+    <div>
+
+      <HomePage />
+      {/* <nav class="main-nav">
```

Comment:
try to remove all the redundant comments here too... just keep the essential comments.